waggle-dance-api/src/test/java/com/hotels/bdp/waggledance/api/model/MetastoreTunnelTest.java

```diff
@@ -72,99 +66,77 @@ public void portTooHigh() {
   @Test
   public void portTooLow() {
     tunnel.setPort(0);
-
     Set<ConstraintViolation<MetastoreTunnel>> violations = validator.validate(tunnel);
-
     assertThat(violations.size(), is(1));
   }
 
   @Test
   public void nullRoute() {
     tunnel.setRoute(null);
-
     Set<ConstraintViolation<MetastoreTunnel>> violations = validator.validate(tunnel);
-
     assertThat(violations.size(), is(1));
   }
 
   @Test
   public void emptyRoute() {
     tunnel.setRoute("");
-
     Set<ConstraintViolation<MetastoreTunnel>> violations = validator.validate(tunnel);
-
     assertThat(violations.size(), is(1));
   }
 
   @Test
   public void blankRoute() {
     tunnel.setRoute(" ");
-
     Set<ConstraintViolation<MetastoreTunnel>> violations = validator.validate(tunnel);
-
     assertThat(violations.size(), is(1));
   }
 
   @Test
   public void nullKnownHosts() {
     tunnel.setKnownHosts(null);
-
     Set<ConstraintViolation<MetastoreTunnel>> violations = validator.validate(tunnel);
-
     assertThat(violations.size(), is(1));
   }
 
   @Test
   public void emptyKnownHosts() {
-    tunnel.setKnownHosts("");
-
+    tunnel.setKnownHosts(" ");
```

Comment:
This is now blank not empty, see the below test and compare.

Reply:
Good spot, thanks!